content/docs/get-started/index.md

```diff
@@ -23,145 +24,30 @@ $ source .venv/bin/activate
 $ pip install pandas scikit-learn mlem[fastapi,heroku]
 ```
 
-</details>
-
-## Saving your ML model
-
-To enable all kinds of productionization scenarios supported by MLEM, we first
-need to save a machine learning model with MLEM.
-
-Let's take a look and run the following Python script:
-
-```py
-from sklearn.datasets import load_iris
-from sklearn.ensemble import RandomForestClassifier
-
-from mlem.api import save
-
-
-data, y = load_iris(return_X_y=True, as_frame=True)
-rf = RandomForestClassifier(
-    n_jobs=2,
-    random_state=42,
-)
-rf.fit(data, y)
-save(
-    rf,
-    "models/rf",
-    sample_data=data,
-)
-```
-
-Here, we load a well-known
-[Iris flower dataset](https://archive.ics.uci.edu/ml/datasets/iris) with
-scikit-learn and train a simple classifier. But instead of pickling the model,
-we save it with MLEM (check out the full list of supported
-[ML frameworks](/doc/object-reference/model)).
+That's it! you're ready to MLEM. It's that simple.
 
-## Productionization
-
-Now MLEM can do the heavy machinery for us, enabling all these scenarios in a
-couple lines of code:
-
-- **[Apply model](/doc/get-started/applying)** - load model in Python or get
-  prediction in command line.
-- **[Serve model](/doc/get-started/serving)** - create a service from your model
-  for online serving.
-- **[Build model](/doc/get-started/building)** - export model into Python
-  packages, Docker images, etc.
-- **[Deploy model](/doc/get-started/deploying)** - deploy your model to Heroku,
-  Sagemaker, Kubernetes, etc.
-
-## Codification
-
-Let's see what we got when we saved a model with MLEM.
+</details>
 
-```cli
-$ tree models/
-models
-├── rf
-└── rf.mlem
-```
+## Scenarios
 
-The model binary was saved to `models/rf`, along with some metadata about it in
-`models/rf.mlem`. We refer to this as to "Codification".
+To explore the basic functionalities and features MLEM provides, we suggest to
+start with these simple scenarios
 
-The `.mlem` file is a bit long, but it contains all the metadata we need to use
-the model later:
+- **[Basic Model Management](/doc/get-started/management)** - Save & load models
+  in Python and get predictions in the command line.
 
-1. Model methods: `predict` and `predict_proba`
-2. Input data schema: describes the data frame (Iris dataset)
-3. Requirements: `sklearn` and `pandas`, with specific versions
+- **[Serving models](/doc/get-started/deploying-and-serving)** - Package your
+  model, create a model server for online serving, and deploy it on the cloud.
```

Comment:
I do like these [Scenarios](https://mlem-ai-simplify-get-st-gwz7i9.herokuapp.com/doc/get-started#scenarios) (use cases? workflows?) more than the current, low-level list of "[Productionization](https://mlem.ai/doc/get-started#productionization)" methods. But I'd still have them after the Saving models section (which lets people start using MLEM right away and hopefully hooked).

Reply:
@jorgeorpinel My initial response here was going to be "but there is symmetry between saving and loading that feels awkward to break"
But after discussing with @shcheklein I want to maybe try and reduce this to 1 workflow/scenarios and significantly shorten it, this is why I'm dilly-dallying with getting back to this for another pass 😆

Reply:
I'm going to push a pass of fixes here and that would be one possible suggestion, and open another PR with an alternative approach (much more condensed)